wireshark-qt: Qt version 6.10.0 requires CMAKE_OSX_DEPLOYMENT_TARGET (11.3) >= 12.0

### Nixpkgs version

- Unstable (25.11)

### Describe the bug

I have Nix installed on my Mac machine, with nix-darwin.

I'm running macOS 26.1 (25B77).'

Nix seems to be currently broken on this machine, due to some issue rebuilding wireshark-qt.

### Steps to reproduce

These are the commands I ran to update my Nix install

```
cd /etc/nix-darwin/
nix flake update
sudo darwin-rebuild switch
```

However, the `darwin-rebuild switch` command seems to fail whilst trying to rebuild wireshark-qt.

### Expected behaviour

wireshark-qt should build without errors.

### Screenshots

_No response_

### Relevant log output

```console
Here is the full output from `sudo darwin-rebuild switch`:

victorhooi@mbp16-personal /e/nix-darwin> sudo darwin-rebuild switch
Password:
building the system configuration...
error: Cannot build '/nix/store/j118lr9hpw01c84k2y8vgcifpyg3wd5h-wireshark-qt-4.6.0.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/2jdzrvx796x1d7yz4lmcwwi0gis7pdf4-wireshark-qt-4.6.0
         /nix/store/d85mys8g6r4drp654hvlkh2w1fq99kn3-wireshark-qt-4.6.0-dev
       Last 25 log lines:
       > -- Could NOT find SBC (missing: SBC_LIBRARY SBC_INCLUDE_DIR)
       > -- Checking for one of the modules 'spandsp'
       > -- Found SPANDSP: /nix/store/xvyn7mhrw3vsz89m3hg34dn221kn3vlh-spandsp-3.0.0/lib/libspandsp.dylib
       > -- Found BCG729: /nix/store/dmkd31kdz4pg9ils77wvpj3al4znfyhj-bcg729-1.1.1/lib/libbcg729.dylib
       > -- Checking for one of the modules 'opencore-amrnb'
       > -- Found AMRNB: /nix/store/04qipmpx4r3jiabjf7irvn17d1mzf75j-opencore-amr-0.1.6/lib/libopencore-amrnb.dylib
       > -- Checking for one of the modules 'libilbc'
       > -- Could NOT find ILBC (missing: ILBC_LIBRARY ILBC_INCLUDE_DIR)
       > -- Checking for one of the modules 'opus'
       > -- Found OPUS: /nix/store/06yzjrnapnc3qadlbrqp0clqpwllg3ik-libopus-1.5.2/lib/libopus.dylib
       > Source locally installed gems is ignoring #<Bundler::StubSpecification name=rbs version=3.4.0 platform=ruby> because it is missing extensions
       > Source locally installed gems is ignoring #<Bundler::StubSpecification name=racc version=1.7.3 platform=ruby> because it is missing extensions
       > Source locally installed gems is ignoring #<Bundler::StubSpecification name=debug version=1.9.2 platform=ruby> because it is missing extensions
       > -- Found Asciidoctor: /nix/store/5jzc0m6idrfq3zf1lrxp31jsc0cc1i5l-asciidoctor-2.0.23/bin/asciidoctor (found suitable version "2.0.23", minimum required is "1.5")
       > -- Could NOT find XSLTPROC (missing: XSLTPROC_EXECUTABLE)
       > -- Could NOT find DOXYGEN (missing: DOXYGEN_EXECUTABLE)
       > -- Checking for one of the modules 'speexdsp'
       > -- Found SpeexDSP: /nix/store/n46j1dvnpp2axlnx30z0drksf34vi6cz-speexdsp-1.2.1/lib/libspeexdsp.dylib
       > -- Looking for gnutls_pkcs11_obj_list_import_url4
       > -- Looking for gnutls_pkcs11_obj_list_import_url4 - found
       > CMake Error at CMakeLists.txt:1660 (message):
       >   Qt version 6.10.0 requires CMAKE_OSX_DEPLOYMENT_TARGET (11.3) >= 12.0
       >
       >
       > -- Configuring incomplete, errors occurred!
       For full logs, run:
         nix log /nix/store/j118lr9hpw01c84k2y8vgcifpyg3wd5h-wireshark-qt-4.6.0.drv
error: Cannot build '/nix/store/ggzsz6mk8l6m6mlr4vj8mviwdfd6z5p7-system-applications.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/vr4wvxrghnrq5rbcjxkbkjqgplv1zgxv-system-applications
error: Cannot build '/nix/store/im0rl9m6cf0w4jb9fr391pl01n8h9ifd-system-path.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/cczgvqvff4zfmlhg8ww9px5yzhcr4gaw-system-path
error: Cannot build '/nix/store/4xivl1f118m379p3yfrc0fks2khr6vf9-darwin-system-25.11.c3c8c9f.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/hx3mp63vp1xha3p4wbcmglymq8xrxhhf-darwin-system-25.11.c3c8c9f
```

### Additional context

Here are the contents of my `/etc/nix-darwin/flake.nix` file:
```
cat /etc/nix-darwin/flake.nix
{
  description = "Example nix-darwin system flake";

  inputs = {
    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
    nix-darwin.url = "github:nix-darwin/nix-darwin/master";
    nix-darwin.inputs.nixpkgs.follows = "nixpkgs";
  };

  outputs = inputs@{ self, nix-darwin, nixpkgs }:
  let
    configuration = { pkgs, ... }: {
      # List packages installed in system profile. To search by name, run:
      # $ nix-env -qaP | grep wget
      environment.systemPackages =
        [ pkgs.vim
          pkgs.uv
          pkgs.fd
          pkgs.bat
          pkgs.numbat
          pkgs.ripgrep
          pkgs.rsync
          pkgs.video-compare
          pkgs.yt-dlp
          pkgs.esptool
          pkgs.ghidra
          pkgs.poppler
          pkgs.nodejs_24
          pkgs.cmake
          pkgs.pkg-config
          pkgs.pdfarranger
          pkgs.nushell
          pkgs.ffmpeg
          pkgs.video-compare
          pkgs.dfu-util
          pkgs.lnav
          pkgs.wireshark
          pkgs.czkawka
          pkgs.ollama
          pkgs.mediainfo
          pkgs.visidata
        ];

      environment.etc.nix-darwin.source = "/Users/victorhooi/dotfiles";
      homebrew = {
        enable = true;
        onActivation.cleanup = "uninstall";
        onActivation.autoUpdate = true;
        onActivation.upgrade = true;

        taps = [];
        brews = [ "cowsay" "esphome"];
        casks = [ "ghostty" "rectangle" "transmit" "stolendata-mpv" "1password" "electrum" "arq" "keka" "microsoft-teams" "microsoft-auto-update" "cleanshot" "todoist-app" "visual-studio-code@insiders" "araxis-merge" "discord" "yaak@beta" "obsidian" "zoom" "zotero" "pycharm" "calibre" "rode-central" "logi-options+" "macwhisper" "imhex" "openmtp" "zed" "qflipper" "adobe-creative-cloud" "protonvpn" "sketch" "autodesk-fusion" "google-chrome@canary" "google-chrome" "vlc" "imazing"];
      };
      # Necessary for using flakes on this system.
      nix.settings.experimental-features = "nix-command flakes";

      # Enable alternative shell support in nix-darwin.
      programs.fish.enable = true;

      # Set Git commit hash for darwin-version.
      system.configurationRevision = self.rev or self.dirtyRev or null;

      # Used for backwards compatibility, please read the changelog before changing.
      # $ darwin-rebuild changelog
      system.stateVersion = 6;

      # The platform the configuration will be used on.
      nixpkgs.hostPlatform = "aarch64-darwin";
      system.primaryUser = "victorhooi";
      users.users.victorhooi.shell = pkgs.fish;
      users.knownUsers = [ "victorhooi" ];
      users.users.victorhooi.uid = 501;
    };
  in
  {
    # Build darwin flake using:
    # $ darwin-rebuild build --flake .#mbp16-personal
    darwinConfigurations."mbp16-personal" = nix-darwin.lib.darwinSystem {
      modules = [ configuration ];
    };
  };
}
```

### System metadata
```
victorhooi@mbp16-personal /e/nix-darwin> nix-shell -p nix-info --run "nix-info -m"
warning: Nix search path entry '/nix/var/nix/profiles/per-user/root/channels' does not exist, ignoring
these 30 paths will be fetched (146.68 MiB download, 1053.08 MiB unpacked):
  /nix/store/ghspqwzhdmzj1rss36lcayyc8mrvparn-apple-sdk-11.3
  /nix/store/vs8p8fpbcws1qkhnnypcf2cdvf0lj1p2-bzip2-1.0.8-bin
  /nix/store/8abvf9rw7jd1smlhb6h6y5ka7srk64fh-cctools-1010.6
  /nix/store/0b51ip5cmpbmiyy1n6bn795wzzwpchrc-cctools-1010.6-libtool
  /nix/store/44wg278ivy2mfqahjbyhbjp3kaw1694q-cctools-binutils-darwin-1010.6
  /nix/store/wspvjcnym763in42ing1dmbdl7yksd9q-cctools-binutils-darwin-wrapper-1010.6
  /nix/store/8xihfndv2sbjnhg8ahm0g7dwa7bgysrw-clang-21.1.1
  /nix/store/qanv9ndpx294a1hyn8rf5xfkvps1q9q1-clang-21.1.1-lib
  /nix/store/l58v977vq4qy58m1bw60bzrpkywizz2b-clang-wrapper-21.1.1
  /nix/store/qx43mfqlmd6nvsz8m5vpjqky0yi1jn5n-diffutils-3.12
  /nix/store/8cs1lzxgymbqzaap6sis8xmyp5n6bwlj-ed-1.22.2
  /nix/store/r8672yyw7gdp7ph0wvw4q6y30c4211a6-expand-response-params
  /nix/store/wz5i5g8qrsjlazwk32by3csgxqpz396b-file-5.45
  /nix/store/fd74ncqjj1nx2f7p0gxsby26xm7njjzl-gnu-config-2024-01-01
  /nix/store/308q7v3aaiw4d1wjijy7xcszlgnfadb6-gnumake-4.4.1
  /nix/store/q4176rdn95rhdb6bhgvf2yr2ihyzvg1p-gnutar-1.35
  /nix/store/y4kxa16fkqbqw85q4v4v0rh9k5z04mjj-ld64-954.16
  /nix/store/q868p6zyh4p8z8zhbl350sk4ypk6njd2-ld64-954.16-lib
  /nix/store/liv6kc1418pdgymwi6h8s9w4gl7h4b5b-libutil-72
  /nix/store/hxhkr0vhmh2i156vh84cs5z1yjarmyg4-llvm-21.1.1
  /nix/store/lhvj9g8v3mm2lhbda6kxksspm4szaw89-llvm-21.1.1-lib
  /nix/store/pv1jfps5ji74ci8j02jzfz0piihh4s73-locale-118
  /nix/store/bsnylm1xz0d3350lzij8yw26wr0qywg0-ncurses-6.5-dev
  /nix/store/bazy1k03nb96pj7ffbkkk96kawn1ybkv-ncurses-6.5-man
  /nix/store/4ykcn26jm4wwzxmi55iqc9zibqh6qi62-patch-2.8
  /nix/store/qi9g4qs1cx0yalsivaa7j5jj6xz5n3af-stdenv-darwin
  /nix/store/fj4fbs4xgfrw1sf2fgg37m3390h6wlpd-update-autotools-gnu-config-scripts-hook
  /nix/store/h1slrd63ph4db7cy25nps36i1d1qmcx0-xar-minimal-501-lib
  /nix/store/a244g3czmlkpqpb7ih8my0hg3cj6cbb3-xcbuild-0.1.1-unstable-2019-11-20-xcrun
  /nix/store/5s96ik2x9amfhyvpxhfc84ixfygr3ra5-xz-5.8.1-bin
copying path '/nix/store/pv1jfps5ji74ci8j02jzfz0piihh4s73-locale-118' from 'https://cache.nixos.org'...
copying path '/nix/store/r8672yyw7gdp7ph0wvw4q6y30c4211a6-expand-response-params' from 'https://cache.nixos.org'...
copying path '/nix/store/liv6kc1418pdgymwi6h8s9w4gl7h4b5b-libutil-72' from 'https://cache.nixos.org'...
copying path '/nix/store/vs8p8fpbcws1qkhnnypcf2cdvf0lj1p2-bzip2-1.0.8-bin' from 'https://cache.nixos.org'...
copying path '/nix/store/wz5i5g8qrsjlazwk32by3csgxqpz396b-file-5.45' from 'https://cache.nixos.org'...
copying path '/nix/store/308q7v3aaiw4d1wjijy7xcszlgnfadb6-gnumake-4.4.1' from 'https://cache.nixos.org'...
copying path '/nix/store/q4176rdn95rhdb6bhgvf2yr2ihyzvg1p-gnutar-1.35' from 'https://cache.nixos.org'...
copying path '/nix/store/qx43mfqlmd6nvsz8m5vpjqky0yi1jn5n-diffutils-3.12' from 'https://cache.nixos.org'...
copying path '/nix/store/bazy1k03nb96pj7ffbkkk96kawn1ybkv-ncurses-6.5-man' from 'https://cache.nixos.org'...
copying path '/nix/store/5s96ik2x9amfhyvpxhfc84ixfygr3ra5-xz-5.8.1-bin' from 'https://cache.nixos.org'...
copying path '/nix/store/8cs1lzxgymbqzaap6sis8xmyp5n6bwlj-ed-1.22.2' from 'https://cache.nixos.org'...
copying path '/nix/store/fd74ncqjj1nx2f7p0gxsby26xm7njjzl-gnu-config-2024-01-01' from 'https://cache.nixos.org'...
copying path '/nix/store/a244g3czmlkpqpb7ih8my0hg3cj6cbb3-xcbuild-0.1.1-unstable-2019-11-20-xcrun' from 'https://cache.nixos.org'...
copying path '/nix/store/q868p6zyh4p8z8zhbl350sk4ypk6njd2-ld64-954.16-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/h1slrd63ph4db7cy25nps36i1d1qmcx0-xar-minimal-501-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/lhvj9g8v3mm2lhbda6kxksspm4szaw89-llvm-21.1.1-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/fj4fbs4xgfrw1sf2fgg37m3390h6wlpd-update-autotools-gnu-config-scripts-hook' from 'https://cache.nixos.org'...
copying path '/nix/store/4ykcn26jm4wwzxmi55iqc9zibqh6qi62-patch-2.8' from 'https://cache.nixos.org'...
copying path '/nix/store/bsnylm1xz0d3350lzij8yw26wr0qywg0-ncurses-6.5-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/qanv9ndpx294a1hyn8rf5xfkvps1q9q1-clang-21.1.1-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/y4kxa16fkqbqw85q4v4v0rh9k5z04mjj-ld64-954.16' from 'https://cache.nixos.org'...
copying path '/nix/store/hxhkr0vhmh2i156vh84cs5z1yjarmyg4-llvm-21.1.1' from 'https://cache.nixos.org'...
copying path '/nix/store/0b51ip5cmpbmiyy1n6bn795wzzwpchrc-cctools-1010.6-libtool' from 'https://cache.nixos.org'...
copying path '/nix/store/8abvf9rw7jd1smlhb6h6y5ka7srk64fh-cctools-1010.6' from 'https://cache.nixos.org'...
copying path '/nix/store/8xihfndv2sbjnhg8ahm0g7dwa7bgysrw-clang-21.1.1' from 'https://cache.nixos.org'...
copying path '/nix/store/44wg278ivy2mfqahjbyhbjp3kaw1694q-cctools-binutils-darwin-1010.6' from 'https://cache.nixos.org'...
copying path '/nix/store/ghspqwzhdmzj1rss36lcayyc8mrvparn-apple-sdk-11.3' from 'https://cache.nixos.org'...
copying path '/nix/store/wspvjcnym763in42ing1dmbdl7yksd9q-cctools-binutils-darwin-wrapper-1010.6' from 'https://cache.nixos.org'...
copying path '/nix/store/l58v977vq4qy58m1bw60bzrpkywizz2b-clang-wrapper-21.1.1' from 'https://cache.nixos.org'...
copying path '/nix/store/qi9g4qs1cx0yalsivaa7j5jj6xz5n3af-stdenv-darwin' from 'https://cache.nixos.org'...
 - system: `"aarch64-darwin"`
 - host os: `Darwin 25.1.0, macOS 26.1`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.31.2`
 - nixpkgs: `/nix/store/pv11fms46b44vv50nvi3l1ckjdlrm6lf-source`
```
### Are you using nix-darwin?

Yes, I am using nix-darwin.

### Notify maintainers

@bjornfor @fpletz 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+bug%22+label%3A%226.topic%3A+darwin%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/networking/sniffers/wireshark/default.nix
+++ b/pkgs/applications/networking/sniffers/wireshark/default.nix
@@ -195,7 +195,7 @@ stdenv.mkDerivation rec {
       flags+=(-change @rpath/"$(basename "$file")" "$file")
     done
 
-    for file in $out/lib/wireshark/extcap/*; do
+    for file in $out/libexec/wireshark/extcap/*; do
       if [ -L "$file" ]; then continue; fi
       echo "$file: fixing dylib references"
       # note that -id does nothing on binaries
@@ -213,7 +213,7 @@ stdenv.mkDerivation rec {
     rm -rf $out/Applications/Wireshark.app/Contents/MacOS/extcap $out/Applications/Wireshark.app/Contents/PlugIns
     mkdir -p $out/Applications/Wireshark.app/Contents/PlugIns
     cp -r $out/lib/wireshark/plugins $out/Applications/Wireshark.app/Contents/PlugIns/wireshark
-    cp -r $out/lib/wireshark/extcap $out/Applications/Wireshark.app/Contents/MacOS/extcap
+    cp -r $out/libexec/wireshark/extcap $out/Applications/Wireshark.app/Contents/MacOS/extcap
   '';
 
   meta = {

--- a/pkgs/applications/networking/sniffers/wireshark/default.nix
+++ b/pkgs/applications/networking/sniffers/wireshark/default.nix
@@ -9,6 +9,7 @@
   buildPackages,
   c-ares,
   cmake,
+  darwinMinVersionHook,
   fixDarwinDylibNames,
   flex,
   gettext,
@@ -147,6 +148,8 @@ stdenv.mkDerivation rec {
   ]
   ++ lib.optionals stdenv.hostPlatform.isDarwin [
     gmp
+    # Required by Qt 6
+    (darwinMinVersionHook "12.0")
   ];
 
   strictDeps = true;

